### PR TITLE
Allow to add ssh elevate credential to target dialog and display elevate credential in details

### PR DESCRIPTION
**Pull request by saberlynx**
**What**:

- Allow creating SSH elevate credentials if SSH credential is set
- If we unset an SSH credential, the SSH elevate credential will be unset as well
- If a target is in use, neither SSH nor SSH elevate credential will be editable
- Can change SSH elevate credential if target is not in use
- Show SSH elevate credentials in targets
- Allow creating new SSH elevate credentials if target is not in use

**Why**:

**How**:

**Checklist**:

Visual confirmation. Will add tests in following PR.

- [ ] Tests
- [x] [CHANGELOG](https://github.com/greenbone/gsa/blob/master/CHANGELOG.md) Entry
- [x] Labels for ports to other branches

### CODE REVIEW

File: gsa/src/web/pages/targets/dialog.js [hunk 1]
```diff
@@ -107,6 +118,10 @@ const NEW_SNMP = {
   types: SNMP_CREDENTIAL_TYPES,
 };
 
+const ElevatePrivilegeText = styled(Layout)`
+  color: ${Theme.darkRed};
```
Comment (sarahd93) [inline gsa/src/web/pages/targets/dialog.js:122]: Actually, this should not be red. I talked about this with @bjoernricks and we realized that was an error in the mockup.

**Review by sarahd93 — CHANGES_REQUESTED** [review]

File: gsa/src/web/pages/targets/dialog.js (hunk 1 above)
Comment (saberlynx) [reply gsa/src/web/pages/targets/dialog.js:122]: Will just regular black work?

Comment (bjoernricks) [reply gsa/src/web/pages/targets/dialog.js:122]: yes just use the standard black.

**Review by bjoernricks — APPROVED** [review]

**Review by sarahd93 — APPROVED** [review]
Awesome work, thank you!

**Pull request merged**